src/content/blog/new-domain-new-discord-same-project.md

```diff
@@ -0,0 +1,59 @@
+---
+title: New Domain, new Discord, same Project!
+description: .xyz is out, .dev is the new black.
+publishDate: 2024-11-27
+ogVariant: winter
+author: louis-escher
+---
+
+Hey all!
+
+Although it's been a while since we last talked, I can assure you that work on StudioCMS has been moving forward. We've been making good progress on 
+`beta.8` and hope to release it towards the end of 2024, although that's more of a guess than a prediction.
+
+You've read the title (or so I hope), so what am I on about here?
+
+Well, you've no doubt seen that we have been [sponsored by Turso](https://studiocms.dev/blog/turso-sponsorship), which were and still are incredibly 
+grateful for. However, it's made us realize that we've outgrown Astrolicious, the community organization who's been helping us nurture our little project. 
```

Comment:
I probably wouldn't phrase it as we've outgrown them. I'd change this to something more like

`However, we're trying to improve our experience as much as possible for both new users, and contributors to the project. At this point we feel we can move away from Astrolicious and go it alone to have full control over our own ecosystem!`